### ISSUE TITLE
Code examples for each usage case

### ISSUE THREAD

**Issue by marcellovictorino** [Netflix/metaflow-nflx-extensions#10]
The readme file is quite thorough and detailed, but it can be too much. 
It would be nice to add some examples, containing the actual metaflow code and requirements file being used, for each to the main use cases.

**Comment by romain-intel** [comment]
That's a good suggestion. I will add a few examples of some of the environments I use to test things out.

**Comment by romain-intel** [comment]
The documentation has some examples of external requirements.txt files taht can work. Are you also looking for stuff inside @conda or @pip in a Metaflow flow?

**Comment by Rachine** [comment]
Yes i would love too to have some short examples, to illustrate the use of main variables 

Thank you for this package

**Comment by romain-intel** [comment]
I'll try to add some more examples.

**Comment by npow** [comment]
Status check (current main): still partially valid. [`docs/conda.md`](https://github.com/Netflix/metaflow-nflx-extensions/blob/main/docs/conda.md) is now ~1145 lines with many inline snippets covering most use cases, but there's still no `examples/` directory with self-contained runnable flow + requirements files per use case as originally requested.
